src/components/panel/panel.scss

```diff
@@ -0,0 +1,20 @@
+.md-panel-container {
+  align-items: center;
+  display: flex;
```

Comment:
I was thinking the other way around, like this:

```
<div class="md-panel-container">
  <div class="md-panel">
    {{ template }}
  </div>
</div>
```

I removed the display:flex from the panel element. Do you think it makes more sense to swap those two classes?